src/components/Table.tsx

```diff
@@ -239,15 +239,15 @@ export function Table(initialState: TableDataType) {
             width: totalWidth,
           },
         })}
-        className={`${c("table noselect cell_size_" + initialState.cellSize)}`}
+        className={`${c("table noselect cell_size_" + initialState.cellSize + (initialState.stickyFirstColumn ? " sticky_first_column" : ""))}`}
```

Comment:
a question. are there no space between those properties?

`"table noselect cell_size_" + initialState.cellSize + (initialState.stickyFirstColumn ? " sticky_first_column" : "")`
if `initialState.cellSize` is size & `(initialState.stickyFirstColumn ? " sticky_first_column" : "")`is B
the result will be `"table noselect cell_size_sizesticky_first_column"`  ?
I underterstand the idea is `"table noselect cell_size_size sticky_first_column"`

Reply:
ooh okay, My mistake =)